src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -208,8 +208,22 @@ runtime::TrtEngineAndContext TensorRTBuilder::BuildEngine(
   CHECK_EQ(engine->getNbBindings(),
            num_input_bindings + network_output_names_.size());
   nvinfer1::IExecutionContext* context = engine->createExecutionContext();
-  return {engine, context, network_input_names_, network_input_is_baked_,
-          network_output_names_};
+  std::vector<runtime::NDArray> device_buffers;
+  for (int i = 0; i < execution_args_.size(); i++) {
```

Comment:
It looks like here we are using the "args" ordering of the inputs, but in `ExecuteEngine` we are still using the "binding_index" ordering. This will cause errors for more complicated models. We need to use a consistent indexing.

Reply:
Done,